discovery-infra/test_infra/tools/assets.py

```diff
@@ -1,14 +1,26 @@
 import json
 import logging
 import os
+import subprocess as sp
+
 from typing import List
 
 from munch import Munch
+from netaddr import IPNetwork, IPAddress
+
 from test_infra import consts, utils
 
 
 class Assets:
     ASSETS_LOCKFILE_DEFAULT_PATH = "/tmp"
+    REQUIRED_FIELDS = (
```

Comment:
We might need this in consts so that we can use the strings elsewhere in the project.